packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -412,23 +412,30 @@ describe('CLI', () => {
     });
   });
 
-  it('generates mnemonic', () => {
-    const out = execCommand('generate-mnemonic');
+  it('generates mnemonic', async () => {
+    const out = execCommand('generate-mnemonic').split('\n');
 
     const explanationInfo = [
       'This mnemonic is created locally on your machine using "ethers.Wallet.createRandom" under the hood.',
       'Make sure to back it up securely, e.g., by writing it down on a piece of paper:',
       '',
-    ]
-      .map((str) => `${str}\n`)
-      .join('');
-
-    expect(out.startsWith(explanationInfo)).toBe(true);
-    const mnemonic = out.split(explanationInfo)[1];
+    ];
+    expect(out.slice(0, 3)).toEqual(explanationInfo);
 
+    const mnemonic = out[3];
     const words = mnemonic.split(' ');
     expect(words).toHaveLength(12);
     words.forEach((word) => expect(word).toMatch(/\w+/));
+
+    const airnodeAddress = await admin.deriveAirnodeAddress(mnemonic);
```

Comment:
@Siegrift We need it `async` for deriving the address here though

Reply:
Sorry, I missed this line and got confused by comment from @andreogle (who I think also missed this line :D )

All good here.